vendor/install-getowner-polyfill.js

```diff
@@ -135,5 +135,17 @@
         };
       }
     });
+
+    Object.defineProperty(_Ember, 'setOwner', {
```

Comment:
Can you move this and guard this in `if (!_Ember.setOwner) {` (it is currently inside of the `if (!_Ember.getOwner) {` guard).